doc/setting_up_sidewalk_environment/setting_up_sdk.rst

```diff
@@ -61,18 +61,9 @@ Follow these steps to download the Sidewalk application for the nRF Connect SDK:
        |___ zephyr
        |___ ...
 
-#. Check out the branch that contains the west manifest file with the Sidewalk group filter:
+#. If you have worked with the Sidewalk application in a previous distribution model (as a manifest repository), check the manifest file.
+   Make sure the manifest path points to :file:`west.yml` inside the :file:`nrf` directory:
 
-   .. code-block:: console
-
-      $ cd nrf
-      $ git checkout main
-      $ git pull
-      $ cd ..
-
-#. Set the Sidewalk application manifest and update.
-
-   Check the current manifest path and make sure it points to :file:`west.yml` inside the :file:`nrf` directory:
 
       .. code-block:: console
 
```

Comment:
Would it be possible to apply through this PR the second comment I left (line 73)?

Reply:
Rephrased the sentence, I hope now it fits better